src/common/sorted-table/index.js

```diff
@@ -220,6 +269,8 @@ export default class SortedTable extends Component {
       () => this.state.activePage,
       this.state.itemsPerPage
     )
+
+    this.state.selectedItems = new Set()
```

Comment:
Does this set contain the items or their ids?

If it contains ids, the name should explicit that.